Assignment4/Assignment4.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+
+function checkError(error) {
+  if(error) {
```

Comment:
space after `if` statement.
Problem Type: Spacing
Example:
`if (error) {` and not `if(error) {`